docs/source/tutorials/basic_renewal_model.qmd

```diff
@@ -158,7 +186,7 @@ The following diagram summarizes how the modules interact via composition; notab
 flowchart TB
     genint["(1) gen_int\n(DetermnisticPMF)"]
     i0["(2) I0\n(InfectionInitializationProcess)"]
-    rt["(3) rt_proc\n(RtRandomWalkProcess)"]
+    rt["(3) rt_proc\n(TransformedRandomVariable)"]
```

Comment:
Maybe we can indicate that it is a transformation of a simple random walk?

Reply:
Addressed in 5869033